config/components/visibility/flowschema.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: flowcontrol.apiserver.k8s.io/v1
```

Comment:
```suggestion
apiVersion: flowcontrol.apiserver.k8s.io/v1beta1
```

This should fix your errors.

Reply:
Yes, it can. But is it ok to use Beta?

Reply:
"flowcontrol.apiserver.k8s.io/v1beta3" is supported between 1.26 - 1.31.
And in v1.32, the version will be removed.

Reply:
> Yes, it can. But is it ok to use Beta?

Is there any reasons why you want to avoid the v1beta3?

Reply:
> "flowcontrol.apiserver.k8s.io/v1beta3" is supported between 1.26 - 1.31.
And in v1.32, the version will be removed.

SGTM

Reply:
Now it's working fine. Thanks!

Reply:
Additionally, could you open an issue to migrate v1beta3 to v1 once we drop the 1.28 supporting?

Reply:
Sure.